soc/arm/st_stm32/stm32wb/soc.h

```diff
@@ -25,7 +25,7 @@
 
 /* Add include for DTS generated information */
 #include <devicetree.h>
-
+#include <stm32wbxx_ll_hsem.h>
```

Comment:
Is that required ?

Reply:
Please discard this one!

Reply:
It is required: if I remove it, I face link issue when building wb.